### DISCUSSION TITLE
Version 0.5.12

### DISCUSSION THREAD

**Discussion by birbilis** [Zoomicon/READCOM_App#222] [Announcements]
Note: ALWAYS DOWNLOAD THE LATEST VERSION FROM [https://github.com/Zoomicon/READCOM_App/releases](https://github.com/Zoomicon/READCOM_App/releases), expanding the "ASSETS" section at the top

There is also a [Windows 10/11 Store version (32-bit)](https://github.com/Zoomicon/READCOM_App/issues/182) that, once installed, can update to newer versions via the "Library" menu of Windows Store app.

For usage information see release notes of [first release](https://github.com/zoomicon/READCOM_App/releases/tag/0.0.1) and see this walk-through [video](https://www.youtube.com/watch?v=0vXp5jUf8cU)

The default document shown when app first launches (or with New button) contains a link to the Tortoise and the Hare Aesop fable story to try.

Alternatively:
* can try downloading manually and opening [that story](https://raw.githubusercontent.com/zoomicon/READCOM_Gallery/master/Gallery/Stories/Aesop%20Fables/The%20Tortoise%20and%20the%20Hare/The%20Tortoise%20and%20the%20Hare.readcom) for testing (right click the link and select to Save As a .readcom file - to use that story's URL in URLActions of StoryItems right click the URL and select Copy, then right-click and Paste in the URLAction editbox at the popup dialog of the ActiveStoryItem in the app).
* can download the whole Gallery in a .ZIP file (extract its Gallery subfolder somewhere on your PC) and load that sample story from the "Stories/Aesop Fables" folder, per the instructions found at [https://github.com/Zoomicon/READCOM_Gallery/](https://github.com/Zoomicon/READCOM_Gallery). The Gallery also contains useful media assets of permissive license at "Assets" subfolder to add to your stories

For a list of Bug fixes and Enhancements in this version see [here](https://github.com/Zoomicon/READCOM_App/issues?q=is%3Aclosed+milestone%3A0.5.12)

The .exe file is compiled for 32-bit Windows (runs on 64-bit Windows too, but for 64-bit you could also download the x64 version that can use more memory)
The .upx.exe file is the same as the 32-bit one, but UPX-compressed (much smaller, might take an instant more to start up and may trigger antivirus false alerts)

For Windows machines with older graphics cards or older graphics drivers with no available driver updates, try holding down SHIFT key when running the application. This will use older Graphics Device Interface (GDI) instead of DirectX on Windows and mention "(Compatibility Mode") on the titlebar of the application.

If you have issues with this version you can try an earlier one, plus please report the issue [here](https://github.com/Zoomicon/READCOM_App/issues) or any suggestions you may have for enhancement.
You may also ask questions and do discussion regarding the READ-COM application at [Discussions](https://github.com/Zoomicon/READCOM_App/discussions).

<hr /><em>This discussion was created from the release <a href='https://github.com/Zoomicon/READCOM_App/releases/tag/0.5.12'>Version 0.5.12</a>.</em>

**Comment by jasa611** [comment]
![image](https://user-images.githubusercontent.com/17861438/168440154-57b1fca0-ea21-4f7c-829b-1d1c3c214af3.png)
This is the error it shows after loading some activities from the  ../gallery/activities

**Reply by birbilis** [reply]
do the .readcom files have a filesize or are they of size 0? There is the chance sometimes the file didn't save and is just a filename with size 0. This means that if you open the .readcom file at a (Unicode UTF8 enabled) text editor (e.g. Notepad++ or even recent Notepad) it may not contain anything (it should start with "object"). If you have other file that is not of size 0, but still says this error, please share it for me to examine

**Comment by jasa611** [comment]
And if I select background storyitem image from start screen, and change by loading from the tool menu (on the right), the APP just crash...

**Reply by birbilis** [reply]
there is a known issue with trying to set an SVG on an ImageStoryItem that has a Bitmap, but the start screen doesn't have a Bitmap, so I'm not sure it's the same one. Also what do you mean crash? Doesn't it show error message and continue? Does it close by itself?

**Comment by jasa611** [comment]
Crash is when app friezes and I have to do ctel alt del to kill the app.

A domingo, 15/05/2022, 18:22, George Birbilis ***@***.***>
escreveu:

**Comment by birbilis** [comment]
can't reproduce this on latest official release version 0.7.1

note that the Illegal Access Error shown when setting via the Options popup dialog the background image of an ImageStoryItem to an SVG vector image file (doesn't occur with bitmap images like JPEG) is not considered a crash since one can just click OK there and continue (and the SVG image has been set ok)

![image](https://user-images.githubusercontent.com/3461504/195040530-de43db01-56b7-4555-be4c-29155ba90293.png)

![image](https://user-images.githubusercontent.com/3461504/195040361-25e2350e-acf3-440a-85ef-a7d7ad26033b.png)

![image](https://user-images.githubusercontent.com/3461504/195040168-bfede700-f63b-446b-9b0a-5e8fc025913d.png)
